lib/http_proxy.c

```diff
@@ -321,7 +321,7 @@ static CURLcode CONNECT(struct Curl_easy *data,
                         host?host:"",
                         data->state.aptr.proxyuserpwd?
                         data->state.aptr.proxyuserpwd:"",
-                        useragent,
+                        useragent?useragent:"",
```

Comment:
Don't we already have a string literal as a fallback option? `User-Agent: curl/7.76.1`?

Reply:
That's just the user-agent the curl tool sets. libcurl itself has no default user-agent!